charts/kubearchive/templates/operator/auth_proxy_service.yaml

```
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: {{ .Values.kubearchive.namespace }}-service
+    app.kubernetes.io/instance: {{ .Values.kubearchive.namespace }}-operator-metrics-service
```

You need to quote it to avoid the error. This will happen case by case until you cover them all, so you need to quote every "mixed" string (interpolation + literal string).

```suggestion
    app.kubernetes.io/instance: "{{ .Values.kubearchive.namespace }}-operator-metrics-service"
```

---

We should be running yamllint on the files *after* they are processed.

---

This is no longer an issue since https://github.com/kubearchive/kubearchive/pull/71 has been committed and CI now runs over the templated files.